cli/user/src/test/java/io/pravega/cli/user/TestUtils.java

```diff
@@ -35,11 +38,60 @@ public static String executeCommand(String inputCommand, InteractiveConfig confi
         CommandArgs args = new CommandArgs(pc.getArgs().getArgs(), config);
         Command cmd = Command.Factory.get(pc.getComponent(), pc.getName(), args);
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        try (PrintStream ps = new PrintStream(baos, true, "UTF-8")) {
+        try (PrintStream ps = new PrintStream(baos, true, StandardCharsets.UTF_8)) {
             assert cmd != null;
```

Comment:
This assertion of assumption should be done right after the `Command` is initialized.